src/aiida/orm/nodes/data/code/installed.py

```diff
@@ -147,6 +151,11 @@ def validate_filepath_executable(self):
                 f'The provided remote absolute path `{self.filepath_executable}` does not exist on the computer.'
             )
 
+        if not user_has_execute:
+            raise exceptions.ValidationError(
+                f'The executable at the remote absolute path `{self.filepath_executable}` is not executable.'
```

Comment:
On some HPCs, FirecREST for example, user on login shell, and user on compute node have different rights. 
User might have executable right only from the compute node. In that case the logic here, may cause confusions.

Reply:
Yeah, that's a valid point. I'm not sure actually how the permissions behave on an _actual_ remote, not just localhost. I'll run some tests here, but I guess it's the same for SSH. Though, difficult to properly test for it in the `tests`, because we just connect to `localhost` there. I'm happy to relax the error message though, as we discussed.